documentation/modules/auxiliary/admin/kerberos/keytab.md

```diff
@@ -63,7 +63,7 @@ Export Kerberos encryption keys stored in the Metasploit database to a keytab fi
 # Secrets dump
 msf6 > use auxiliary/gather/windows_secrets_dump
 msf6 auxiliary(gather/windows_secrets_dump) > run smbuser=Administrator smbpass=p4$$w0rd rhosts=192.168.123.13
-... ommitted ...
+... omitted ...
```

Comment:
This was me, the shame

Reply:
I fixed a few of my own as well. happens to everyone!